Sources/SpeziLicense/Resources/Localizable.xcstrings

```diff
@@ -0,0 +1,70 @@
+{
+  "sourceLanguage" : "en",
+  "strings" : {
+    "License Information" : {
+      "extractionState" : "manual",
```

Comment:
We should ensure that the extraction states for all strings are automatic so they are extracted if needed.

Reply:
Somehow, I couldn't figure out how to do that 🤔

Reply:
@NikolaiMadlener You can find this setting in the inspector sidebar:
<img width="1624" alt="Screenshot 2024-04-01 at 1 59 09 AM" src="https://github.com/StanfordSpezi/SpeziLicense/assets/28656495/6bd5e185-7e5a-43e2-8f28-0ab4e1b71c0a">